src/Controls/src/Core/Label/Label.Android.cs

```diff
@@ -39,6 +39,7 @@ private protected override void OnHandlerChangedCore()
 		public static void MapText(ILabelHandler handler, Label label)
 		{
 			Platform.TextViewExtensions.UpdateText(handler.PlatformView, label);
+			InvalidateMeasureIfNecessary(label);
 		}
```

Comment:
Do we know why is now required to invalidate measure here? https://github.com/dotnet/maui/blob/bd614274484c7fa0fd7fac008c551932613b75b3/src/Controls/src/Core/Label/Label.cs#L422 Is InvalidateMeasureIfLabelSizeable, invoked with any modification in the Label bounds (text, character spacing, etc) working?

Reply:
The initial problem here is that the order of events was happening like this

BP fires property changed
InvalidateMeasure Fires too early for CV to catch and remeasure
Mapperr Runs (this is what changes the text)

If you look at Label.cs you'll see where i've basically just moved the invalidation call so now it fires in this order

BP fires property changed
Mapperr Runs (this is what changes the text)
InvalidateMeasure Fires now with the text changed